webapp/.gitignore

```diff
@@ -7,6 +7,10 @@
 
 # testing
 /coverage
+/test-results/
+/playwright-report/
+/blob-report/
+/playwright/.cache/
```

Comment:
I see that the HTML reporter by default writes to `playwright-report` but why do we ignore `/test-results/`, `/blob-report/` and `/playwright/.cache/`?

 https://playwright.dev/docs/test-reporters#html-reporter

Reply:
Was thinking that files generated from tests runners should be kept out of version control. Like we would keep build folders out.

Maybe Ive misunderstood something?